playbooks/automation/check-patch.yml

```diff
@@ -2,4 +2,4 @@
 - import_playbook: "{{ playbook_dir }}/../utils/configure-std-ci-repos.yml"
   when: std_ci_yum_repos is defined
 - import_playbook: "{{ playbook_dir }}/../cluster/{{ cluster_type | default('openshift') }}/config.yml"
-- import_playbook: "{{ playbook_dir }}/../components/install-kubevirt-release.yml"
+- import_playbook: "{{ playbook_dir }}/playbooks/kubevirt-resources.yml"
```

Comment:
@rthallisey I think that the path is incorrect:

`ERROR! Unable to retrieve file contents
Could not find or access '/home/jenkins/workspace/kubevirt_kubevirt-ansible_standard-check-pr/kubevirt-ansible/playbooks/automation/playbooks/kubevirt-resources.yml'
`

Reply:
I made a mistake rebasing. Should be ```"{{ playbook_dir }}/playbooks/kubevirt.yml"```